TwoPage/src/main/java/com/microsoft/device/display/samples/twopage/PagerAdapter.kt

```diff
@@ -3,20 +3,31 @@
  * Licensed under the MIT License.
  *
  */
+
 package com.microsoft.device.display.samples.twopage
 
 import android.util.SparseArray
+import androidx.core.util.forEach
 import androidx.fragment.app.Fragment
 import androidx.fragment.app.FragmentManager
 import androidx.fragment.app.FragmentPagerAdapter
+import com.microsoft.device.display.samples.twopage.fragments.ScrollingContent
 
 internal class PagerAdapter(
-    fm: FragmentManager?,
+    fragmentManager: FragmentManager,
```

Comment:
love it! these small things, that somehow you understand but that maybe requires a bit of time to guess what it was are good candidates to be changed. `fragmentManager` is a very better name, more descriptive and clear.